apps/remix-ide/src/app/plugins/parser/services/code-parser-compiler.ts

```diff
@@ -142,6 +142,12 @@ export default class CodeParserCompiler {
         this.compiler.set('runs', state.runs)
         this.compiler.set('useFileConfiguration', true)
         this.compiler.set('compilerRetriggerMode', CompilerRetriggerMode.retrigger)
+
+        if (await this.plugin.call('fileManager', 'exists','remappings.txt')) {
+          const remappings = await this.plugin.call('fileManager', 'readFile','remappings.txt')
+          this.compiler.set('remappings', remappings.split('\n').filter(Boolean))
+        }
```

Comment:
Please add an else section to reset the state which helps when remapping file is deleted.